cfme/base/__init__.py

```diff
@@ -59,6 +61,32 @@ def zone(self):
     def slave_servers(self):
         return self.zone.collections.servers.filter({'slave': True}).all()
 
+    @property
```

Comment:
Anyway these could not be duplicated?   That is it really looks like the same code in both cases.

Reply:
Server uses `sid`, Zone uses `id`, and Region uses `number` attributes, and the collection referenced is different.  Possibly a settingsmixin class, but each would still need a property to know what attribute to reference for the `id` field.  Doesn't make sense in the context of BaseEntity to put it there.

Reply:
It does indeed work, and they are in fact different settings dicts: http://pastebin.test.redhat.com/563692